src/refill-relayers.ts

```diff
@@ -0,0 +1,139 @@
+import { createPublicClient, createWalletClient, http, parseEther } from "viem";
+import { celo, celoAlfajores } from "viem/chains";
+import { privateKeyToAccount } from "viem/accounts";
+import * as fs from "fs";
+import * as path from "path";
+import { config } from "./config";
+import { deriveRelayerAccount } from "./utils";
+import getSecret from "./get-secret";
+
+const MIN_BALANCE_THRESHOLD = 5;
+const TRANSFER_AMOUNT = 50;
+
+const networks = {
+  mainnet: {
+    name: "mainnet",
+    chain: celo,
+    rpcUrl: "https://forno.celo.org",
+    relayerAddressesFile: "infra/relayer_addresses.json",
```

Comment:
nit: it's the same for both envs so no point in an env-specific config here imo

Reply:
removed